### PR TITLE
Create `free-courses-ta.md` with several courses in Tamil

### PR DESCRIPTION
**Pull request by GH05T-HUNTER5**
We are trying to make new Tamil Tutorial Playlist

### CODE REVIEW

**Comment by davorpa** [comment]
Tamil locale registration is done at #7000

**Review by davorpa — COMMENTED** [review]
Please solve linter error(s) raised by [the GitHub Actions task run #7820856998](https://github.com/EbookFoundation/free-programming-books/runs/7820856998?check_suite_focus=true)

```
/courses/free-courses-ta.md
     7:1-7:8  warning  Incorrect number of blank lines between headings  blank-lines-1-0-2  remark-lint
    7:8-11:1  warning  Remove empty section: "PHP"                       no-empty-sections  remark-lint
  11:1-11:12  warning  Remove empty section: "Python"                    no-empty-sections  remark-lint
```

**Comment by GH05T-HUNTER5** [comment]
> Please solve linter error(s) raised by [the GitHub Actions task run #7820856998](https://github.com/EbookFoundation/free-programming-books/runs/7820856998?check_suite_focus=true)
> 
> ```
> /courses/free-courses-ta.md
>      7:1-7:8  warning  Incorrect number of blank lines between headings  blank-lines-1-0-2  remark-lint
>     7:8-11:1  warning  Remove empty section: "PHP"                       no-empty-sections  remark-lint
>   11:1-11:12  warning  Remove empty section: "Python"                    no-empty-sections  remark-lint
> ```

Ok Please Wait

**Comment by GH05T-HUNTER5** [comment]
First Create
free-courses-ta.md

**Review by davorpa — COMMENTED** [review]
Please solve error(s) raised by [the GitHub Actions `free-programming-books-lint #5125` task run](https://github.com/EbookFoundation/free-programming-books/runs/7823661329?check_suite_focus=true)

```
/courses/free-courses-ta.md
  49:1-50:165  warning  Alphabetical ordering: swap l.50 and l.49  alphabetize-lists  remark-lint
  61:1-62:115  warning  Alphabetical ordering: swap l.62 and l.61  alphabetize-lists  remark-lint
```

File: courses/free-courses-ta.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+### PHP
+
+* [PHP for beginners In Tamil](https://youtube.com/playlist?list=PL4unWLKFsZfcq_D-sEy0pR4Sl_yipy6Jt) - Tutor Joe's Stanley
+* [php Tutorial In Tamil](https://youtube.com/playlist?list=PL4unWLKFsZfdrMitLmm8N-idlYQkSCvT9) - Tutor Joe's Stanley
+
+
+### Python
+
+* [PYTHON TUTORIAL IN TAMIL](https://youtube.com/playlist?list=PLIFRUdRwOM0_hcLruKbsHWnU5P2uLBgsp) - SANTRA TECHSPOT
+* [Python in Tamil for Beginners](https://youtube.com/playlist?list=PLA2UBjeRwle3OLO3qmXTbmCvuTlqhHRVb) - GURUKULA
```
Comment (davorpa) [inline courses/free-courses-ta.md:62]: Alphabetize swapping records 61-62 

```suggestion
* [Python in Tamil for Beginners](https://youtube.com/playlist?list=PLA2UBjeRwle3OLO3qmXTbmCvuTlqhHRVb) - GURUKULA
* [PYTHON TUTORIAL IN TAMIL](https://youtube.com/playlist?list=PLIFRUdRwOM0_hcLruKbsHWnU5P2uLBgsp) - SANTRA TECHSPOT
```

File: courses/free-courses-ta.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+### Java
+
+* [Learn Java in Tamil](https://youtube.com/playlist?list=PLYM2_EX_xVvVXm005Gt5unmqW6GGMjHxa) - Logic First Tamil
+
+
+### JavaScript
+
+* [JavaScript in Tamil](https://youtube.com/playlist?list=PL73Obo20O_7ihsIM5K-hHYPrcqkkdQcLa) - CyberDude Networks Pvt. Ltd.
+* [javascript Full Course in Tamil - in Depth Javascript Tutorials in Tamil](https://youtube.com/playlist?list=PLpYn3LR7eQI3hjh129Bkqkw7onut28hTK) - Tamil Developer
```
Comment (davorpa) [inline courses/free-courses-ta.md:50]: Alphabetize swapping records 49-50

```suggestion
* [Javascript Full Course in Tamil - in Depth Javascript Tutorials in Tamil](https://youtube.com/playlist?list=PLpYn3LR7eQI3hjh129Bkqkw7onut28hTK) - Tamil Developer
* [JavaScript in Tamil](https://youtube.com/playlist?list=PL73Obo20O_7ihsIM5K-hHYPrcqkkdQcLa) - CyberDude Networks Pvt. Ltd.
```

File: courses/free-courses-ta.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+### <a id="cpp"></a>C++
+
+* [C++ Programming in Tamil](https://youtube.com/playlist?list=PLWbtDrDnmTHBPD-Dt5BJi7iP11x6UvgU0) - CS in Tamil
+
+
+### HTML and CSS
+
+* [CSS in Tamil](https://youtube.com/playlist?list=PL73Obo20O_7gGv4cLEOoqTF8_m8rPKyQh) - CyberDude Networks Pvt. Ltd.
+* [HTML Course in Tamil](https://youtube.com/playlist?list=PL73Obo20O_7gcXt0cfQA14jey8zavtKAq) - CyberDude Networks Pvt. Ltd.
+* [Learn HTML in Tamil | HTML Tutorials in Tamil | Beginner to Advanced](https://youtube.com/playlist?list=PLpYn3LR7eQI2trAr1z1lmvYGAQfsbllY5) - Tamil Developer
```
Comment (davorpa) [inline courses/free-courses-ta.md:39]: Do a Markdown escape for pipes present in resource titles (see #5176 for a informative reason)

```suggestion
* [Learn HTML in Tamil \| HTML Tutorials in Tamil \| Beginner to Advanced](https://youtube.com/playlist?list=PLpYn3LR7eQI2trAr1z1lmvYGAQfsbllY5) - Tamil Developer
```

File: courses/free-courses-ta.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+### JavaScript
+
+* [JavaScript in Tamil](https://youtube.com/playlist?list=PL73Obo20O_7ihsIM5K-hHYPrcqkkdQcLa) - CyberDude Networks Pvt. Ltd.
+* [javascript Full Course in Tamil - in Depth Javascript Tutorials in Tamil](https://youtube.com/playlist?list=PLpYn3LR7eQI3hjh129Bkqkw7onut28hTK) - Tamil Developer
+
+
+### PHP
+
+* [PHP for beginners In Tamil](https://youtube.com/playlist?list=PL4unWLKFsZfcq_D-sEy0pR4Sl_yipy6Jt) - Tutor Joe's Stanley
+* [php Tutorial In Tamil](https://youtube.com/playlist?list=PL4unWLKFsZfdrMitLmm8N-idlYQkSCvT9) - Tutor Joe's Stanley
```
Comment (davorpa) [inline courses/free-courses-ta.md:56]: Copy titles as same as original source

```suggestion
* [PHP Tutorial In Tamil](https://youtube.com/playlist?list=PL4unWLKFsZfdrMitLmm8N-idlYQkSCvT9) - Tutor Joe's Stanley
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Nice resources :rocket: but needs some changes before get merged :arrow_forward:

Please, take your time to commit following suggestions using a web interface :wink:

**Comment by GH05T-HUNTER5** [comment]
Nice

**Comment by GH05T-HUNTER5** [comment]
👍

**Comment by davorpa** [comment]
Please, don't resolve suggestions without committing any changes. It turns review process into headache.

File: courses/free-courses-ta.md (hunk 3 above)
Comment (davorpa) [reply courses/free-courses-ta.md]: Resolved by commit a0e2f8203e6c91cc2fea737a703a653e2133e166

File: courses/free-courses-ta.md (hunk 1 above)
Comment (davorpa) [reply courses/free-courses-ta.md]: Resolved by commit 4a4220baba7e816cb29cc11ee4f9f2b833aa89cc

File: courses/free-courses-ta.md (hunk 2 above)
Comment (davorpa) [reply courses/free-courses-ta.md]: Fixed by commit 4a4220baba7e816cb29cc11ee4f9f2b833aa89cc

File: courses/free-courses-ta.md (hunk 4 above)
Comment (davorpa) [reply courses/free-courses-ta.md]: Resolved by commit 43172c07c7e96c4197eff03b4d01ff15b320ab7e

**Review by davorpa — APPROVED** [review]
LGTM

**Comment by GH05T-HUNTER5** [comment]
> LGTM

😀

**Comment by GH05T-HUNTER5** [comment]
> > LGTM
> 
> 😀

Please Add This

File: courses/free-courses-ta.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+### PHP
+
+* [PHP for beginners In Tamil](https://youtube.com/playlist?list=PL4unWLKFsZfcq_D-sEy0pR4Sl_yipy6Jt) - Tutor Joe's Stanley
+* [PHP Tutorial In Tamil](https://youtube.com/playlist?list=PL4unWLKFsZfdrMitLmm8N-idlYQkSCvT9) - Tutor Joe's Stanley
+
+
+### Python
+
+* [Python in Tamil for Beginners](https://youtube.com/playlist?list=PLA2UBjeRwle3OLO3qmXTbmCvuTlqhHRVb) - GURUKULA
+* [PYTHON TUTORIAL IN TAMIL](https://youtube.com/playlist?list=PLIFRUdRwOM0_hcLruKbsHWnU5P2uLBgsp) - SANTRA TECHSPOT
```
Comment (eshellman) [inline courses/free-courses-ta.md:62]: From CONTRIBUTING.md 

> - No ALLCAPS titles. Usually title case is appropriate, but when doubt use the capitalization from the source

Comment (GH05T-HUNTER5) [reply courses/free-courses-ta.md:62]: What do you mean sir?

Comment (eshellman) [reply courses/free-courses-ta.md:62]: Please read CONTRIBUTING.md guidelines!

Comment (GH05T-HUNTER5) [reply courses/free-courses-ta.md:62]: Ok

Comment (GH05T-HUNTER5) [reply courses/free-courses-ta.md:62]: > Please read CONTRIBUTING.md guidelines!

Sir how to solve this

**Comment by GH05T-HUNTER5** [comment]
> LGTM

Dude how to solve this

**Comment by GH05T-HUNTER5** [comment]
> > LGTM
> 
> Dude how to solve this

@davorpa @davorpa 
Let me close these

File: courses/free-courses-ta.md (hunk 5 above)
Comment (davorpa) [reply courses/free-courses-ta.md:62]: Source is in ALLCAPS but... here the title-case patch...

```suggestion
* [Python Tutorial in Tamil](https://youtube.com/playlist?list=PLIFRUdRwOM0_hcLruKbsHWnU5P2uLBgsp) - SANTRA TECHSPOT
```

**Review by davorpa — APPROVED** [review]

File: courses/free-courses-ta.md (hunk 5 above)
Comment (davorpa) [reply courses/free-courses-ta.md]: Resolved by a813fdf71ee1a0f09978e1e781d0cdb6ac2da885

**Review by eshellman — APPROVED** [review]

**Pull request merged**